app/code/Magento/MediaGalleryUi/Test/Mftf/Test/AdminMediaGalleryDefaultViewWithoutFiltersTest.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminMediaGalleryDefaultViewWithoutFiltersTest">
+        <annotations>
+            <features value="AdminMediaGalleryDefaultViewWithoutFiltersTest"/>
+            <useCaseId value="https://github.com/magento/adobe-stock-integration/issues/1813"/>
+            <title value="User shouldn't see applied filters if media gallery switched to Default View"/>
+            <description value="No filters should be applied if Default View selected"/>
```

Comment:
Please add `stories` and `testCaseId` to annotation - otherwise MFTF tests are going to fail on validation stage.

Please contact @chalov-anton to identify which `testCaseId` should be used